lib/kotkowo_web/components/drawers.ex

```diff
@@ -12,39 +12,48 @@ defmodule KotkowoWeb.Components.Drawers do
   attr :title, :string, required: true
   attr :class, :string, default: ""
   attr :folded, :boolean, default: true
-  attr :class_when_hidden, :string, default: "hidden"
+  attr :class_when_hidden, :string, default: "hidden xl:block"
   attr :title_class, :string, default: "font-bold xl:text-xl"
   slot :inner_block, required: true, doc: "Content when unfolded"
 
   def drawer(assigns) do
     ~H"""
-    <div>
-      <div
-        class={
-          classes(["border rounded-xl xl:w-[757px] flex flex-col cursor-pointer select-none", @class])
-        }
-        x-data={~c"{folded: #{@folded}}"}
-      >
-        <div
-          class="p-6 flex justify-between align-center"
-          x-bind:class="!folded && 'pb-3'"
-          x-on:click="folded = !folded"
-        >
-          <span class={@title_class}><%= @title %></span>
-          <template x-if="folded">
-            <.icon name="chevron_down" class="w-4 lg:w-5 h-3 lg:h-4 my-auto" />
-          </template>
-          <template x-if="!folded">
-            <.icon name="chevron_up" class="w-4 lg:w-5 h-3 lg:h-4 my-auto" />
-          </template>
+    <div
+      x-data={"{isXl: window.innerWidth >= 1280, folded: #{@folded}}"}
+      x-on:resize.window="isXl = window.innerWidth >= 1280"
```

Comment:
Use this instead.
https://github.com/kotkowo/kotkowo/blob/ac88026a8f693f581a6c47f62e1547091c6e0c53/lib/kotkowo_web/components/sections.ex#L93-L95